tests/testthat/test-utilities-simulation-results.R

```diff
@@ -17,7 +17,7 @@ test_that("It throws an error when no valid simulation results are provided", {
 })
 
 test_that("It returns an array of NA if specific result is not found and stopIfNotFound = FALSE", {
-  res <- getOutputValues(individualResults, "NoPath", stopIfNotFound = FALSE)
+  res <- getOutputValues(simulationResults = individualResults, quantitiesOrPaths = "NoPath", stopIfNotFound = FALSE)

```

Comment:
Let's try to always use argument names in test to capture errors if the signature of the method changes.